lib/rules/property-no-unknown/index.js

```diff
@@ -67,12 +70,16 @@ function rule(actual, options) {
 				return;
 			}
 
-			const { selector } = decl.parent;
+			const { selector, name, type } = decl.parent;
```

Comment:
We need to check all the at-rules up to the root, not just the parent. 

For example, given `"supports"` the following should be accepted:

```css
@supports(display: grid) {
  @media (min-width: 10px) {
    a {
      my-property: 1
    }
  }
}
```